src/Components/Blueprints/BlueprintActionsMenu.tsx

```diff
@@ -59,11 +67,39 @@ export const BlueprintActionsMenu: React.FunctionComponent<
         >
           Edit details
         </DropdownItem>
-        <DropdownItem>Download blueprint (.json)</DropdownItem>
+        <DropdownItem
+          onClick={() => {
+            if (blueprintName && blueprintDetails) {
+              handleExportBlueprint(blueprintName, blueprintDetails);
+            }
+          }}
+        >
+          Download blueprint (.json) <BetaLabel />
+        </DropdownItem>
         <DropdownItem onClick={() => setShowDeleteModal(true)}>
           Delete blueprint
         </DropdownItem>
       </DropdownList>
     </Dropdown>
   );
 };
+
+async function handleExportBlueprint(
+  blueprintName: string,
+  blueprint: BlueprintResponse
+) {
+  const opts = {
+    suggestedName: blueprintName.replace(/\s/g, '_').toLowerCase() + '.json',
+    types: [
+      {
+        description: 'Text file',
+        accept: { 'text/plain': ['.json'] },
+      },
+    ],
+  };
+
+  const fileHandle = await (window as any).showSaveFilePicker(opts);
```

Comment:
The linter fails because of this, do we have a solution other than ignoring the linter?